site/app/models/OfficeHoursQueueViewer.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace app\models;
+
+use app\libraries\Core;
+use app\libraries\DateUtils;
+use app\models\OfficeHoursQueueViewer;
+
+class OfficeHoursQueueViewer extends AbstractModel {
+
+
+    //If you want to see more details on the status codes see DatabaseQueries.php
```

Comment:
I would like to see those status codes be enums within this model (or as a new model or something) and then all checks against specific numbers use those enum values, which makes it easier to follow the logic over time, and we are not relying on magic numbers anywhere.